version.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# As seen in <https://pagure.io/waiverdb/blob/master/f/rpmbuild.sh#_16>
+
+set -ex
+
+if [ "$(git tag | wc -l)" -eq 0 ] ; then
```

Comment:
This if statement will never execute because we do use tags. Perhaps you should consider removing it.